### PR TITLE
Change the default file placement

### PR DESCRIPTION
**Pull request by mgrabul**
This is update of the dependencies that caused problems while including them in other projects and also update to the default location of the openapi-docs.json file. The file is saved at a location so while the build process in jenkins it will be picked up and persisted automatically to appropriate s3

### CODE REVIEW

File: example/src/main/resources/application.properties [hunk 1]
```diff
@@ -1 +1 @@
-openapi.service.output = build/apiDoc/api-docs.json
\ No newline at end of file
+openapi.service.output = api-docs.json
```
Comment (lanwen) [inline example/src/main/resources/application.properties]: please revert this change, doesn't make any sense to assign it to default value

File: .gitignore [hunk 2]
```diff
@@ -30,3 +30,4 @@ testgen/.gradle
 out/
 
 OpenApiDocsTest.java
+/example/api-docs.json
```
Comment (lanwen) [inline .gitignore]: please remove, build dir is excluded already

Comment (mgrabul) [reply .gitignore]: done, now the file is saved in the build directory so no entry in the gitignore is needed

File: example/src/main/resources/application.properties (hunk 1 above)
Comment (mgrabul) [reply example/src/main/resources/application.properties]: done

**Review by lanwen — APPROVED** [review]
